NuRadioReco/modules/trigger/radiant_aux_trigger.py

```diff
@@ -0,0 +1,173 @@
+        in order to trigger the station. The number of coincidences specifies the number of channels that need to have triggered within the coincidence window.
+        
+        Parameters
+        ----------
+        evt: Event
+            Event to run the module on
+        station: Station
+            Station to run the module on
+        det: Detector
+            The detector description
+        number_coincidences: int
+            number of channels that are required in coincidence to trigger a station
+        triggered_channels: array of ints or None
+            channels ids that are triggered on, if None trigger will run on all channels
+        coinc_window: float
+            time window in which number_coincidences channels need to trigger
+        trigger_name: string
+            a unique name of this particular trigger
+        """
+        t = time.time()  # absolute time of system
+
+        self.sampling_rate = station.get_channel(det.get_channel_ids(station.get_id())[0]).get_sampling_rate()
+        dt = 1. / self.sampling_rate
+
+        triggered_bins_channels = []
+        channels_that_passed_trigger = []
+
+        if triggered_channels is None:  # caveat: all channels start at the same time
+            for channel in station.iter_channels():
+                channel_trace_start_time = channel.get_trace_start_time()
+                break
+        else:
+            channel_trace_start_time = station.get_channel(triggered_channels[0]).get_trace_start_time()
+
+        for channel in station.iter_channels():
+            channel_id = channel.get_id()
+            if triggered_channels is not None and channel_id not in triggered_channels:
+                logger.debug("skipping channel {}".format(channel_id))
+                continue
+
+            if channel.get_trace_start_time() != channel_trace_start_time:
+                logger.warning('Channel has a trace_start_time that differs from '
+                               '        the other channels. The trigger simulator may not work properly')
+
+            (triggered_bins, threshold) = self.get_trigger_response(channel.get_trace(), threshold_sigma, int_window)
+            triggered_bins_channels.append(triggered_bins)
+
+            if True in triggered_bins:
+                channels_that_passed_trigger.append(channel.get_id())
+            
+            channel.set_parameter(chp.radiant_aux_trigger_threshold, threshold)
+        
+        has_triggered, triggered_bins, triggered_times = get_majority_logic(triggered_bins_channels,
+                                                                            number_coincidences, coinc_window, dt)
+        trigger = RadiantAUXTrigger(trigger_name, threshold_sigma, int_window, number_coincidences, coinc_window, triggered_channels)
+        trigger.set_triggered_channels(channels_that_passed_trigger)
```

Comment:
If you give `None` to your module, then this trigger also gets `None`. Maybe it would be better to build a list of channels that do consider for the trigger during this script (maybe you can do it in the for loop at L132)

Reply:
Maybe at the beginning, you can right away do

```
considered_channels = copy(triggered_channels)
if triggered_channels is None:
    considered_channels = list(station.get_channel_ids())
```
and then use `considered_channels` everywhere here

Reply:
Thanks, that is nice 👍